linuxkit/linuxkit
# wireguard: update to 0.0.20191212

**- What I did**

Bumped wireguard to the latest release.

**- How I did it**

Edited the URL and `sha256`

**- How to verify it**

Unsure -- I don't use wireguard myself. This was requested by a user of Docker Desktop and looked simple. I'm hoping that a CI run will be enough to check.

**- Description for the changelog**

- Bump wireguard to 0.0.20191212
